common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -29,30 +29,42 @@ killall_program() {
     done
 }
 
+prplmesh_platform_db_init() {
+    management_mode=${1-Multi-AP-Controller-and-Agent}
+    operating_mode=${2-Gateway}
+
+    mkdir -p /tmp/@PLATFORM_DB_PATH_TEMP@/
+    {
+        echo -e "management_mode=${management_mode}"
+        echo -e "operating_mode=${operating_mode}"
+        echo -e "wired_backhaul=1"
+    } > /tmp/@PLATFORM_DB_PATH_TEMP@/prplmesh_platform_db
```

Comment:
It *happens* to work if `@PLATFORM_DB_PATH_TEMP@` is substituted by nothing, but I don't think it's intended this way...